### PR TITLE
Add more tests for layers in 3 d/4 d

### LINKED ISSUE #302: Test current version before releasing
I'm not sure if all use cases were properly tested. It would be cool if someone volunteers for testing them before #281 , as the tool is downloaded and used by quite some folks.

* [x] 2D labels
* [x] 3D labels
* [x] 2D + time
* [x] 3D + time
* [x] points
* [x] surface
* [x] points + time ?  (bf09d794596de073a993f5e8c942782bb1a42ff3)
* [x] surface + time ?  (9cf67049d4222163e2f9c08f4c46778eddb52729)

For the last tests, example data might be necessary (see also #301 )

Thanks!

### PR DESCRIPTION
**Pull request by zoccoler**
This adds extra tests and should fix #302 

@jo-mueller , could you take a look?

### CODE REVIEW

File: napari_clusters_plotter/_tests/test_plotter.py [hunk 1]
```diff
@@ -79,6 +79,278 @@ def test_plotting(make_napari_viewer):
...
+    z_slices = 3
+    n_timepoints = 2
+    n_labels = get_labels_array().max()
+    data_3d_timelapse = np.array(
+        [
+            np.roll(np.stack([get_labels_array() for _ in range(z_slices)]), t, axis=1)
+            for t in range(n_timepoints)
+        ]
+    )
+    # Create some random features
```
Comment (jo-mueller) [inline napari_clusters_plotter/_tests/test_plotter.py:202]:
```suggestion
    assert len(data_3d_timelapse) == 4
    # Create some random features
```

File: napari_clusters_plotter/_tests/test_plotter.py [hunk 2]
```diff
@@ -79,6 +79,278 @@ def test_plotting(make_napari_viewer):
...
+        data_2d_timelapse,
+        properties={
+            "label": label_column,
+            "feature1": feature1,
+            "feature2": feature2,
+            "annotations": annotations,
+            "frame": frame,
+        },
+    )
+
```
Comment (jo-mueller) [inline napari_clusters_plotter/_tests/test_plotter.py:163]:
```suggestion
    assert len(data_3d_timelapse) == 4
```

**Review by jo-mueller — APPROVED** [review]
Just some minor suggestions to make sure that we are always handling 4D data

File: napari_clusters_plotter/_tests/test_plotter.py (hunk 1 above)
Comment (zoccoler) [reply napari_clusters_plotter/_tests/test_plotter.py:202]: I like that!

File: napari_clusters_plotter/_tests/test_plotter.py (hunk 2 above)
Comment (zoccoler) [reply napari_clusters_plotter/_tests/test_plotter.py:163]: Here it should be `data_2d_timelapse`. I will fix that.

**Review by jo-mueller — APPROVED** [review]

**Pull request merged**